### PR TITLE
[16.0] [MIG] sale_order_report_without_price

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

File: sale_order_report_without_price/report/sale_report_templates.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_saleorder_document_without_price"
+        inherit_id="sale.report_saleorder_document"
+        primary="True"
+        priority="99"
+    >
+        <xpath expr="//th[@name='th_priceunit']" position="replace" />
```
Comment (rousseldenis) [inline sale_order_report_without_price/report/sale_report_templates.xml]: @marielejeune Wasn't it possible to avoid replaces ?

Comment (marielejeune) [reply sale_order_report_without_price/report/sale_report_templates.xml]: Indeed. Added `t-if=False` attribute instead.

**Review by rousseldenis — APPROVED** [review]

**Review by hildickethan — APPROVED** [review]

**Pull request merged**